package-lock.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Are you missing a `package.json` file too?

Reply:
It's not necessary to send `package.json` to version control if you have a `package-lock.json`

Reply:
Shouldn't both be committed? Else dependency management might be tricky.

Also it looks like there's no deps actually defined here.